File: src/main/java/programmers/training/day01/l181949/Solution.java
package programmers.training.day01.l181949;

import java.util.Scanner;

public class Solution {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String str = sc.next();

        StringBuilder sb = new StringBuilder();
        for (char c : str.toCharArray()) {
            if (Character.isLowerCase(c)) {
                sb.append(Character.toUpperCase(c));
            } else {
                sb.append(Character.toLowerCase(c));
            }
        }
        System.out.println(sb);
    }
}

File: src/main/java/programmers/training/day01/l181949/Solution.kt
package programmers.training.day01.l181949

fun main(args: Array<String>) {
    val str = readln()

    val sb = StringBuilder()
    for (c in str) {
        sb.append(
            if (c.isLowerCase()) c.uppercaseChar() else c.lowercaseChar()
        )
    }
    println(sb)
}